ListingManager.Tests/FileManagerTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace ListingManager.Tests
+{
+    [TestClass]
+    public class FileManagerTests
+    {
+
+        [TestMethod]
+        public void GetChapterNumber()
+        {
+            string chapter1 = "Chapter01";
+            string chapter42 = "asdasdsad/asd/asd/asd/Chapter42";
+
+            Assert.IsTrue(FileManager.GetFolderChapterNumber(chapter1) == 1);
+            Assert.IsTrue(FileManager.GetFolderChapterNumber(chapter42) == 42);
```

Comment:
Use Assert.AreEqual(obj expected, obj actual) for comparing two objects

Reply:
gotcha, but in this case it's an int == an int...

Reply:
I see I have fallen for one of the classic blunders... Never use Assert.IsTrue when clarity is on the line!